### DISCUSSION TITLE
How to manually change learning rate?

### DISCUSSION THREAD

**Discussion by drscotthawley** [Lightning-AI/pytorch-lightning#13190] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Hi, I see descriptions of learning rate schedulers, etc.  What I want to do is to just manually set the learning rate during a run, something  "optim.something.something.lr = x".    

Would that be just by going down to "raw PyTorch", e.g. like

```Python
optimizer.param_groups[0]['lr'] = new_value
```
...? Thanks!

**Comment by drscotthawley** [comment]
Before I posted I searched for similar posts but didn't find anything. Now I'm seeing this post which seems to describe doing the same thing: 
https://github.com/PyTorchLightning/pytorch-lightning/discussions/7169

**Reply by akihironitta** [reply]
@drscotthawley I'd suggest following the comment there: https://github.com/PyTorchLightning/pytorch-lightning/discussions/7169#discussioncomment-647121
